akka-docs/src/test/java/jdocs/actor/ActorDocTest.java

```diff
@@ -412,7 +412,7 @@ public void usePatternsGracefulStop() throws Exception {
     //#gracefulStop
     try {
       CompletionStage<Boolean> stopped =
-        gracefulStop(actorRef, Duration.create(5, TimeUnit.SECONDS), Manager.SHUTDOWN);
+        gracefulStop(actorRef, java.time.Duration.ofSeconds(5), Manager.SHUTDOWN);
```

Comment:
There are still a lot of things in this test (so in the docs) that use scala's Duration e.g. setting receiveTimeout.

We ofc don't need to get to them all in this PR

Reply:
Yes, we will not close the ticket #24646, please add notes there. I think receiveTimeout was already mentioned.